index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Mobile Version</title>
+</head>
+<body>
+    
+    <div class="toolbar">
+        <div class="colm">My Logo</div>
+        <div class="coli">
+            <img src="Union.png" alt="">
+        </div>
+
+    </div>
+    <div class="headline-section">
+        <div class ="col1"><header>
+            <h1 class="name">I'm Alaa Shaheen Glad to see you!</h1>
+        </header></div>
+        <div class="col1">
+    
+    <p>I’m a software developer! I can help you build a product , 
+        feature or website Look through some of my work and experience!
+         If you like what you see and 
+        have a project you need coded, don’t hestiate to contact me.</p>
+    </div>
+    </div>
+    
+    <img class="img1" src="images/Ellipse 3.png" alt="">
+    <img class="img2" src="images/Ellipse 11.png" alt="">
+    <img class="img3" src="images/Ellipse 5.png" alt="">
+    <img class="img4" src="images/Ellipse 8.png" alt="">
+    <img class="img5" src="images/Rectangle 7.png" alt="">
+    <img class="img6" src="images/Rectangle 4.png" alt="">
```

Comment:
You did a good job however for this section you have to do some changes to match the Figma design:

- [ ] Use [flexbox](https://www.w3schools.com/css/css3_flexbox.asp) to align the elements of the section.
- [ ] Kindly replace the container to `section` instead of `div` on line 19.
- [ ] Please make sure that the section has the margins (top, bottom, left, right) and fonts defined in the design.
- [ ] Kindly, for the list of icons use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).

Your design:
![feed1](https://user-images.githubusercontent.com/53347966/176934738-d8c9ecbc-f79a-477f-9d34-7f1925cc6a73.JPG)

Figma design
![feedfigma](https://user-images.githubusercontent.com/53347966/176934750-ca78ee34-8f8d-44b0-842e-0794385d78b0.JPG)

Reply:
@algerina 
 Kindly, for the list of icons use a ul tag to group them, and li per icon; each icon has a link (a tag).
 i try to do this it show me a dots because its list !